apps/music/style/music.css

```diff
@@ -266,12 +290,15 @@ body.picker-mode #title-player {
   overflow-y: auto;
   background: -moz-linear-gradient(center top, #303030 0%, #000000 100%) repeat scroll 0 0 transparent;
   z-index: 10;
-  transition: transform 300ms ease;
+  transition: transform 300ms ease, visibility 300ms;
+  visibility: hidden;
 }
 
 .view {
   width: 100%;
   height: auto;
+  visibility: hidden;
+  transition: visibility 300ms;
```

Comment:
I checked the spec and it does said the `visibility` property supports transition, but do we actually see it when we apply it? maybe we just simply show/hide the views by changing the `visibility` style without transition, and possibly gain some fps.